### ISSUE TITLE
Switch off semantic correction and other DOM corrections

### ISSUE THREAD

**Issue by vasotelvi** [JiHong88/SunEditor#720]
Hi!
Is it possible to switch off semantic correction and other DOM corrections?

**Comment by JiHong88** [comment]
@vasotelvi Sorry I don't understand what I mean.
Could you please explain again?

**Comment by vasotelvi** [comment]
I have to see raw text data in SunEditor. In raw text I have 
```
<div class='message'>
<h4>Heading</h4>
<p>Paragraph</p>
</div>
```
When I initialize SunEditor, I have this semantic correction – heading extracted outside of div container
```
<h4>Heading</h4>

<div class='message'>
<p>Paragraph</p>
</div>
```
And every line break `\n` in text generates paragraphs wrapping text from line break to line break
```
Text \n
Text
```
transform to
```
<p>Text</p>
<p>Text</p>
```
Hope, I explained enough.

**Comment by JiHong88** [comment]
You can use the "__se__tag" or "__se__format_xxx" class.

"__se__tag" refer: 
https://github.com/JiHong88/SunEditor/issues/397
https://github.com/JiHong88/SunEditor/issues/594#issuecomment-758641952

"__se__format_xxx" refer:
https://github.com/JiHong88/SunEditor/issues/412#issuecomment-648065274

simple ex)
```
<div class='message __se__format__range_message'>
  <h4>Heading</h4>
  <p>Paragraph</p>
</div>
```

**Comment by vasotelvi** [comment]
Unfortunately I can't modify and add extra class in raw text to solve this inconvenience before I initialize SunEditor for textareas. 😪

**Comment by JiHong88** [comment]
I don't understand..
The contents of the initial "text area" must be entered directly.

**Comment by anthomus** [comment]
I believe what @vasotelvi is suggesting and I agree is to have a config setting that would disable semantic corrections for the entire component vs having to put this in a class attribute.  With this setting, we would not want Suneditor to **reorder** tags or rewrite tags if they are valid HTML/XML.  It is OK to remove tags not in the "whitelist", but not to restructure the HTML.

Reasons:

1. Users of the application containing the Suneditor will not know to use these special class attributes, and therefore as the application programmer we want to be able to control whether to allow corrections or not.  (Generally I can't imagine a use case for reordering HTML that was pasted or entered.)

2. Users may have existing HTML snippets that they are pasting in that require certain HTML structure, and reordering elements may not change the appearance in the Suneditor, but may  change the behavior of the HTML where used with CSS selectors.

**Comment by anthomus** [comment]
Would it be easy to add a config setting that would achieve the same behavior as __se__tag for the entire component in the 2.x branch?

**Comment by JiHong88** [comment]
@anthomus Yes you are right, In code view mode, I will modify the format format so that it does not check duplicates.

"format" is an important prerequisite for the current editor function.
Custom issues related to "format" like this issue will be fixed in v3.0.0.

**Comment by JiHong88** [comment]
The [2.41.0](https://github.com/JiHong88/SunEditor/releases/tag/2.41.0) version has been updated.
Thank you.
